manticore/platforms/linux.py

```diff
@@ -3316,6 +3316,44 @@ def _interp_total_size(interp):
         last = load_segs[-1]
         return last.header.p_vaddr + last.header.p_memsz
 
+    @classmethod
+    def implemented_syscalls(cls) -> Iterable[str]:
+        """
+        Get a listing of all concretely implemented system calls for Linux. This
+        does not include whether a symbolic version exists. To get that listing,
+        use the SLinux.implemented_syscalls() method.
+        """
+        import inspect
+
+        return (
+            x[0]
+            for x in inspect.getmembers(cls, predicate=inspect.isfunction)
```

Comment:
Extremely minor nit: In situations like this, I'd prefer unpacking over indexing since you're going to reuse the piece of data in multiple places, and `for (name, _) in inspect.getmembers(cls, predicate=inspect.isfunction)` makes it a little bit more clear what `getmembers` returns.

Reply:
Yes... Not sure what I was thinking using just `x` lol. Thank you!